Makefile

```diff
@@ -36,7 +36,8 @@ TARGETS := $(MAKE_TARGETS) $(CMD_TARGETS)
 
 DOCKER_TARGETS := $(patsubst %,docker-%, $(TARGETS))
 .PHONY: $(TARGETS) $(DOCKER_TARGETS)
-DOCKERFILE_DEVEL := $(CURDIR)/deployments/container/Dockerfile.devel
+DOCKERFILE_DEVEL := "images/devel/Dockerfile"
```

Comment:
Is there supposed to be a file added for this?:
```
+ DOCKERFILE_DEVEL := "images/devel/Dockerfile"
```

Reply:
no, this is the location of the file at the `K8S_TEST_INFRA` location